### ISSUE TITLE
New Release Request

### ISSUE THREAD

**Issue by rvikart** [lidatong/dataclasses-json#378]
It seems that there are some new changes that are not part of the latest release. Is there an ETA on when a new release with the latest changes will be created?

**Comment by lidatong** [comment]
Yes, automated releases through CI is a great idea. Can create a new issue for that and link this one
